README.rst

```diff
@@ -1,15 +1,15 @@
-Getting Started
+Golioth Zephyr SDK
 ***************
 
-Using with Zephyr RTOS
+Getting Started
 ======================
```

Comment:
Line endorsement length should be the same as the section title length. Same in other places.